src/test/java/io/bf2/kafka/authorizer/VerifiableAppenderExtension.java

```diff
@@ -13,48 +13,66 @@
 import org.junit.jupiter.api.extension.ParameterContext;
 import org.junit.jupiter.api.extension.ParameterResolutionException;
 import org.junit.jupiter.api.extension.ParameterResolver;
+import org.junit.jupiter.api.extension.TestInstancePostProcessor;
+import org.junit.platform.commons.util.StringUtils;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
 
 //based on/inspired by https://www.dontpanicblog.co.uk/2018/01/15/test-log4j-with-junit/
 // Which is in turn based on the MIT licensed https://github.com/hotblac/voicexmlriot/blob/voicexmlriot-0.1.0/src/test/java/org/vxmlriot/jvoicexml/junit/LogAppenderResource.java
 public class VerifiableAppenderExtension implements BeforeEachCallback, AfterEachCallback, ParameterResolver {
     public static final String LOG_EVENTS_STORE_KEY = "logEvents";
-    public static final String APPENDER_NAME = "VerifiableAppender";
+    public final String appenderName;
     private final ExtensionContext.Namespace namespace;
 
     public VerifiableAppenderExtension() {
         namespace = ExtensionContext.Namespace.create(VerifiableAppenderExtension.class);
+        appenderName = "VerifiableAppender-" + Instant.now();
     }
 
     @Override
     public void afterEach(ExtensionContext extensionContext) {
         extensionContext.getStore(namespace).remove(LOG_EVENTS_STORE_KEY, List.class);
```

Comment:
Should we also remove `LoggerName` from store namespace after each test?

Reply:
Yes.